app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -302,6 +304,12 @@ public static VideoDetailFragment getInstance(final int serviceId, final String
         return instance;
     }
 
+    public static VideoDetailFragment getInstanceCollapsed() {
```

Comment:
I'd call this `newCollapsedInstance`

Reply:
`new` means that can be another instance, but it's not possible here. Instance can not be collapsed. So I just named it `getInstanceInCollapsedState`. Ok?